Formula/m/malcontent.rb

```diff
@@ -22,6 +22,15 @@ def install
     system "go", "build", *std_go_args(ldflags: "-s -w -X main.BuildVersion=#{version}", output: bin/"mal"), "./cmd/mal"
   end
 
+  def caveats
+    <<~EOS
+      malcontent embeds Yara rules in order to scan files and
+      certain AV or anti-malware software may detect malcontent as malicious.
+      Programs that leverage Yara will generally see other programs that also use Yara as malicious
+      due to their respective strings looking for problematic behavior.
+    EOS
+  end
```

Comment:
Thanks for the PR @egibs! Caveats are generally for issues specifically related to Homebrew packaging [^1], so judging from https://github.com/chainguard-dev/malcontent/issues/727 this may not be a good fit?

[^1]: https://docs.brew.sh/Formula-Cookbook#caveats

Reply:
Totally fair! We’ll rely on that messaging instead.